website/project/issue-pr-grading.rst

```diff
@@ -0,0 +1,418 @@
+staging area to integrate code (and uncover potential integration
+issues) before publishing code to ``main``.
+
+This means that, to get code into ChiGame, you must make pull request
+to get code merged into the ``dev`` branch (and the instructor and TAs
+will decide when to merge that code into ``main``).
+
+Because the ``dev`` branch should contain stable code, pull requests to
+``dev`` must be carefully reviewed before they are approved. If you want
+to merge code into ``dev``, your code must be reviewed by (1) someone in
+your team and (2) your senior developer. Please note that the senior
+developer will only do their code review once your teammate has completed
+their review. Additionally, when your code affects code
+owned by another team, you must have an additional reviewer from each
+team whose code is affected.
+
+A pull request should contain a relatively self-contained piece of work.
+In fact, it is not uncommon for a task (or, at most, 2-3 tasks) to
+result in a pull request. We encourage you to touch base with your
+senior developer before creating your first pull requests, to determine
+whether the work you want to merge merits a pull request (or, similarly,
+whether it needs to be broken up into multiple pull requests).
+
+Like a task, a pull request should have a descriptive title, a
+descriptive summary, updates on the progress of the pull request
+(especially since changes may need to be done based on the code
+reviews), and a closing statement. The assignees for the pull request
+must be everyone who contributed to the code that is being merged.
+
+Before a pull request is approved, the merge being merged must be
+"up to date" with ``dev``, meaning that any changes that happened
+in ``dev`` while the pull request was being reviewed must be merged
+into the pull request's branch. It is sometimes not uncommon for
+a pull request to take a few days to get approved (because the review
+takes a while, or because you realize there are more changes you need to
+make in that pull request); when that happens, you want to get into
+the habit of merging from ``dev`` frequently, and not just right before
+the pull request is approved.
+
+Finally, take into account that the final outcome of a pull request could be for
+that pull request to be denied. If this happens, it will not impact your
+score (the effect will be the same as if you had never made the pull
+request to begin with).
+
+Grading
+-------
+
+Each issue and pull request will be scored on the ESNU scale
+(following the rubric we describe further below). Additionally,
+an issue or a pull request may be determined to be a "minor"
+issue or pull request (e.g., if you create a pull request to fix
+a small bug that only requires changing a few lines of code).
+
+Based on this, you will receive the following number of points
+for a completed issue or pull request:
+
+.. cssclass:: table-bordered
+
++---+-----------------+-----------------+
```

Comment:
If an issue has an associated pull request, will students receive points for both? Ex. earning an E on the issue and PR will give 30 points. If so, I think this should be clarified

Reply:
Yes, they would receive points for both. I've added a sentence to clarify this.